test.py

```diff
@@ -2,4 +2,15 @@
 import numpy as np
 
 def detectar_nulos(df):
-    return df.isnull().sum()
\ No newline at end of file
+    return df.isnull().sum()
+
+def detectar_nulos(df):
+    return df.isnull().sum()
+    
+def calcular_promedios(df):
+    return df.mean()
+
+def calcular_tipos(df):
+    for col in df.iteritems():
+        print(col.dtypes)
```

Comment:
De esta forma quedaria megor, sugiero cambiarla.
```suggestion
def calcular_tipos(df):
        print(df.dtypes)
```